src/apis/router_console_dc_tracker.erl

```diff
@@ -180,6 +180,8 @@ handle_info(
     {blockchain_event, {add_block, _BlockHash, _Syncing, _Ledger}},
     #state{chain = undefined} = State
 ) ->
+    lager:info("got block ~p with not chain", [_BlockHash]),
+    erlang:send_after(500, self(), post_init),
```

Comment:
We add a post_init message in `init`. 
Isn't this just going to add another check every 500ms for every block we receive without a chain?
I think it's highly unlikely, but also a little unnecessary.